### ISSUE TITLE
Activity bar focus improvments

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#97610]
As suggested by @joaomoreno
Since we have improved the status bar focus, the following should be done in the activity bar

1. Clicking anywhere in the activity bar should focus the whole activity bar
2. Once actvitiy bar is focused it should have a small blue border
3. Pressing up / down focuses the activity bar items like today
4. Pressing Home / End focuses the first / last activity bar item
5. Pressing `esc` resets focus to the whole activity bar

@sandy081 @sbatten let me know what you think. I can also look into this if you think it makes sense.

fyi @jvesouza @bpasero

**Comment by bpasero** [comment]
I am not a big fan of all the blue outlines we have for containers that can have focus. If we want to be consistent, wouldn't we need to do the same for the panel-bar, any action bar, possible the editor tabs parent, breadcrumbs?

**Comment by isidorn** [comment]
Well currently it is a bit funky that you can not just click on the empty area in the activity bar.

I believe all our widgets should decide the following:
1) I can be focused and thus I render blue outline
2) When I am focused I auto focus first element - like the action bar

**Comment by bpasero** [comment]
@isidorn I think a component should only show a blue outline and support being focused if it does not automatically pass focus to the first element. That is where the tree is special because certain actions target the root when nothing is focused.

But for places such as the activity bar or even status bar I would argue that moving focus in should automatically focus the first thing in there.

**Comment by isidorn** [comment]
Status bar is a bit special because when focused Screen REaders should read the overall content so users get an impression of the whole content. Thus focus is first on the full widget.

As for actvitiy bar we can say it belongs to catogory 2., thus it should auto focus first item when clicked upon.

**Comment by bpasero** [comment]
But today you cannot click somewhere into the status bar to focus it, you need to use this F6 thing right? Are you suggesting to change it for status bar too?

**Comment by isidorn** [comment]
Clicking in the status bar empty area focuses it now.

**Comment by bpasero** [comment]
That is not really discoverable though because there is no focus indication.

**Comment by isidorn** [comment]
There is focus, just blue on blue looks funky and you do not see it with the default colors.

**Comment by alex19EP** [comment]
I am absolutely for it. I was surprised today when the home and end did not work.

**Comment by isidorn** [comment]
Once we change the focus behvaior of the activity bar we shuold align the status bar as well.

For example click anywhere would focus the first entry (or last focused entry)

**Comment by isidorn** [comment]
We have changed how the actionBar focus works and activity bar respects that now.
No further changes planned here
